src/components/Partners.astro

```diff
@@ -0,0 +1,101 @@
+    format('woff');
+    font-weight: normal;
+    font-style: normal;
+    }
```

Comment:
Duplicate of [this](https://github.com/tul-ftims-students-council/Delta/blob/master/src/styles/lemonmilk.css)